pages.es/linux/rig.md

```diff
@@ -0,0 +1,24 @@
+# rig
+
+> Utilidad para generar un nombre, apellido, calle y número, junto a ubicación geográfica consistente (un conjunto válido de ciudad, estado y código postal).
+> Más información: <https://manpages.ubuntu.com/manpages/focal/man6/rig.6.html>.
+
+- Muestra un nombre aleatorio (masculino o femenino) y una dirección:
+
+`rig`
+
+- Muestra un nombre [m]asculino, (o [f]emenino) aleatorio y una dirección:
+
+`rig -{{m|f}}`
+
+- Usa archivos de datos de un directorio específico (por defecto es `/usr/share/rig`):
+
+`rig -d {{ruta/al/directorio}}`
+
+- Especifica el número de identidades a generar:
+
+`rig -c {{número}}`
```

Comment:
These are variable names, so we prefer to keep the accents out.
```suggestion
`rig -c {{numero}}`
```

Reply:
We do? Maybe should normalize this throughout the pages... 🤔 

I'd be happy to do that, I we determine for sure that accents **should not** be in variable names.

Reply:
I do not really see the need for unaccented words in variable names. However, in programming contexts this is good practice, as accents can mess things up. I guess this could be analogous.

Reply:
@bl-ue the nouns in German starts with caps and we normalized that to strict `snake_case` recently.

Reply:
Let me know the final decision on this matter, whether I should commit this or not. I also have a couple more translations ready to be pushed which depend on it.

Reply:
Yeah I remember that, I looked at that just yesterday.  @sbrl have you any opinion?

Reply:
I don't think it really matters either way. I guess "variable name" can be somewhat subjective, but l'd suggest it's best to follow convention.

I'm not sure that this is something that could be easily standardised though, because determining what constitutes a variable name would be a somewhat subjective process though.

Reply:
It is more natural to have variables without accents IMO. But I don't really have a strong opinion nor a big preference.

Reply:
Okay then, lets remove the accents @goznalo-git 👍🏻